test/libsolidity/SolidityExpressionCompiler.cpp

```diff
@@ -635,7 +635,7 @@ BOOST_AUTO_TEST_CASE(selfbalance)
 
 	bytes code = compileFirstExpression(sourceCode, {}, {});
 
-	if (solidity::test::CommonOptions::get().evmVersion() == EVMVersion::istanbul())
+	if (solidity::test::CommonOptions::get().evmVersion() >= EVMVersion::istanbul())
```

Comment:
SHouldn't this actually use `hasSelfBalance()` instead?

Reply:
Well, they are the same, aren't they? Anyway I'll change this.